<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTodo</class>
 <widget class="QMainWindow" name="MyTodo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyTodo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Sitka Small</family>
          <pointsize>22</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#243B4A;
color:#87BCDE;</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="text">
         <string>My To-Do List</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>Task List</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="1" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Priority      Title</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPushButton" name="Calendar">
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>6</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">boder:none;</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="0" colspan="2">
             <widget class="QPushButton" name="AddNew">
              <property name="font">
               <font>
                <family>Terminal</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton
{background-color:#243B4A;
color:#87BCDE;
border-radius:10px;
rotation: 90px;
}

QPushButton:pressed
{background-color:#1e303d;
color:#78b3da;
border-radius:10px;}
</string>
              </property>
              <property name="text">
               <string>Add New</string>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QListWidget" name="TasksList">
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">background-color:#cab2eb;
font: 15px;
border-radius:10px;</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" rowspan="2" colspan="2">
             <widget class="QLabel" name="label_3">
              <property name="styleSheet">
               <string notr="true">background-color:#243B4A;
color:#87BCDE;</string>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="trashButton">
              <property name="font">
               <font>
                <family>Terminal</family>
                <pointsize>24</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>29</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="DoneButton">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{background-color:#243B4A;
color:#87BCDE;
border-radius:10px;}

QPushButton:pressed
{background-color:#1e303d;
color:#78b3da;
border-radius:10px;}
</string>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="BackButton">
      <property name="font">
       <font>
        <family>Terminal</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{background-color:#243B4A;
color:#87BCDE;
border-radius:10px;}

QPushButton:pressed
{background-color:#1e303d;
color:#78b3da;
border-radius:10px;}
</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>trashButton</sender>
   <signal>clicked()</signal>
   <receiver>TasksList</receiver>
   <slot>selectAll()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>229</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
